Source/Core/loadCRN.js

```diff
@@ -0,0 +1,84 @@
+/*global define*/
+define([
+    './CompressedTextureBuffer',
+    './defined',
+    './DeveloperError',
+    './loadArrayBuffer',
+    './TaskProcessor',
+    '../ThirdParty/when'
+], function(
+    CompressedTextureBuffer,
+    defined,
+    DeveloperError,
+    loadArrayBuffer,
+    TaskProcessor,
+    when) {
+    'use strict';
+
+    var transcodeTaskProcessor = new TaskProcessor('transcodeCRNToDXT', Number.POSITIVE_INFINITY);
```

Comment:
Are you sure about `Number.POSITIVE_INFINITY`?

We have some bigger work to do on parallelism, but, in the meantime, what is a reasonable default?